content-service/src/main/java/io/meeds/news/activity/processor/ActivityNewsProcessor.java

```diff
@@ -93,6 +97,22 @@ public void processActivity(ExoSocialActivity activity) {
         LOG.warn("Error retrieving news with id {}", activity.getTemplateParams().get("newsId"), e);
       }
       activity.getLinkedProcessedEntities().put("news", news);
+      try {
+        String newsId = news.getId();
+        List<String> articleLanguages = newsService.getArticleLanguages(newsId, false);
```

Comment:
In the implem of this method .getArticleLanguages, withDrafts is always true, to be verified

Reply:
https://github.com/Meeds-io/content/blob/feature/experience/content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java#L1138

Reply:
update the description of apiResponses for the rest service getAvailableTranslationLanguages